### ISSUE TITLE
Characters in Doomheim are still showing up in former corp as current members

### ISSUE THREAD

**Issue by CanuckLiquor** [zKillboard/evewho#14]
Hi there, it seems Doomheim isn't being recognized as a corporation. 

Toons that are in Doomheim are still showing up as members in their last corp 

Here is an example of two toons from a  single corp.

![image](https://user-images.githubusercontent.com/36280806/81244054-584ad680-8fdf-11ea-8b6f-1c1e430db650.png)

![image](https://user-images.githubusercontent.com/36280806/81244226-cd1e1080-8fdf-11ea-9d98-00f2369e7a28.png)

![image](https://user-images.githubusercontent.com/36280806/81244168-a3fd8000-8fdf-11ea-8171-1c0aa3b56a31.png)

 that third toon is still in the player corp as a reference

**Comment by cvweiss** [comment]
What is the URL of the third screenshot showing the character still in the corporation?

**Comment by CanuckLiquor** [comment]
Yeah sorry it was hard grabbing a  full screenshot of the page. It was meant to provide you with a reference point  for AoDWildcat Darine showing them in their current corp.
